draft-ietf-rats-corim.md

```diff
@@ -1491,36 +1491,97 @@ and how the verifier matches a Reference Value against the Accepted Claims Set
 
-[^issue]: Need to describe how to match conditional endorsements. Tracked at https://github.com/ietf-rats-wg/draft-ietf-rats-corim/issues/80
+> *TODO: Can we find a better term than the placeholder "group"? If so then I think we should replace all mentions of "group" with the new term.*
 
 #### Matching all Reference Values in a group against the Accepted Claims Set {#sec-match-all-ref-vals}
 
-If all Reference Values in a group match entries in the Accepted Claims Set
-then all Endorsements in the group are added to the Accepted Claims Set
-(see {{sec-add-to-acs}}). {{sec-match-one-ref-val}} describes how one
+If a group contains at least one `reference-triple-record` and at least one
+`endorsed-triple-record` then the Reference Values contained in the former
+are used to decide whether to add Endorsed Values in the latter to the
+Accepted Claims Set.
+
+The Verifier SHALL try to match all Reference Values contained within all
+`reference-triple-record`s within the group against Evidence and Endorsements
+in the Accepted Claims Set. {{sec-match-one-ref-val}} describes how each
 Reference Value is matched against the Accepted Claims Set.
 
+If all Reference Values in a group match entries in the Accepted Claims Set
+then all Endorsed Values from `endorsed-triple-record`s in the group are added
+to the Accepted Claims Set (see {{sec-add-to-acs}}).
+
 If any Reference Value in a group does not match the Accepted Claims Set then
-all Endorsements in the group are silently ignored.
+all `endorsed-triple-record`s in the group are silently ignored.
+
+Each group is processed independently of other groups. If `reference-triple-record`s
+in a group do not match the Accepted Claims Set then this does not affect
+the processing of other groups.
+
+#### Matching a Conditional Endorsement Triple against the Accepted Claims Set
+
+A Conditional Endorsement Triple {{sec-comid-triple-cond-end}} is an efficient
+encoding of a Reference Value and an Endorsed Value.
+
+The triple's `stateful-environment-record` field contains both the `environment-map` and `measurement-map` parts of the Reference Value.
+
+The Endorsed Value is encoded using the same `environment-map`, plus the
+`measurement-value-map` field within the Conditional Endorsement Triple.
+
+For each Conditional Endorsement Triple, the verifier SHALL attempt to match the
+Reference Value against entries in the Accepted Claims Set (see {{sec-match-one-ref-val}}).
+If it matches then the verifier SHALL add the Endorsed value to the
+Accepted Claims Set (see {{sec-add-to-acs}}).
+
+>> *TODO: Is a Conditional Endorsement Triple affected by `reference-triple-record`s
+in the same group? I think not. Does this need to be mentioned?*
+
+#### Matching a Conditional Endorsement Series Triple against the Accepted Claims Set
+
+A Conditional Endorsement Series Triple {{sec-comid-triple-cond-series}} is an efficient
+encoding of a series of alternative Reference Values, each with a corresponding
```

Comment:
I think this is misstated. The series claims are endorsed values, of which one in the series is selected for inclusion in the ACS. The RVs are not a "series" but a stateful environment.

Reply:
The is the introductory description of what the triple does, IMHO it should describe the rest of the section in informative words so that a reader knows what is coming next.
I used the word "series" because the verifier processes them in order.
I haven't found a non-circular definition of "stateful environment" so I don't think that is a good term to use instead. It might be if we define it appropriately. Perhaps what I called "Reference Value" is what you think of as "stateful environment"?